synapse/module_api/__init__.py

```diff
@@ -748,6 +748,36 @@ def record_user_external_id(
             )
         )
 
+    async def create_login_token(
+        self,
+        user_id: str,
+        duration_in_ms: int = (2 * 60 * 1000),
+        auth_provider_id: Optional[str] = None,
+        auth_provider_session_id: Optional[str] = None,
+    ) -> str:
+        """Create a login token suitable for m.login.token authentication
+
+        Added in Synapse v1.69.0.
```

Comment:
1.69.0rc1 has now shipped I'm afraid, so this should now be `1.70.0` (with the removal scheduled for `1.72.0`).

Reply:
I re-targeted the PR to the `release-v1.69` branch to hopefully squeeze it into the next RC

Reply:
Have you discussed this with someone else in the backend team? Usually we try not to expand the scope of a release once the first RC has shipped (especially since we don't usually do more than 1 RC unless there's a bug that's preventing people from testing usefully). So I'd prefer if we target 1.70 and shift the deprecation/removal schedule by a release if possible.

Reply:
Discussed it internally, 1.69 is fine actually.